pages/common/return.md

```diff
@@ -0,0 +1,12 @@
+# return
+
+> Exit out of a function or a script if run with `source`.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#index-return>.
+
+- Exit out of a function prematurely:
+
+`{{func_name}}() { {{echo "This is reached"}}; return; {{echo "This is not"}}; }`
+
+- Specify function return value:
```

Comment:
```suggestion
- Specify the function's return value:
```

Reply:
I feel my wording is not wrong and it feels more punctual. I'll leave it as is unless you have convincing arguments.

Reply:
At least the `the` is needed in order to have a correct English sentence. 

The `'s` is personal preference. Writing `Specify the function return value` implies that `function return value` is a commonly used name. I personally think that `Specify the function's return value` reads more smoothly, but I'll approve both if you like the version without the `'s` better.

Reply:
The pains of being an ESL.